blog/2024/11-30-android-tv-18/index.mdx

```diff
@@ -0,0 +1,135 @@
+---
+
+Version 0.18 of our Jellyfin for Android TV app is ready! Exciting features like lyrics and media segments are now available, along with
+significant improvements to subtitles and more.
+
+{/* truncate */}
+
+This release marks another milestone in our journey to delivering the best media app on Android TV. Jellyfin 10.10 or newer is required. A
+complete list of changes, including their respective pull requests, can be found on
+[GitHub](https://github.com/jellyfin/jellyfin-androidtv/releases/tag/v0.18.0). I've listed a subset of all changes below.
+
+### Skipping intros, outros and more with media segments
+
+Media segments are a new feature added in Jellyfin 10.10. A segment provides metadata for specific parts of a video or audio file. This can
+be used to mark an intro in your episode or the credits of a movie. The Android TV app can now use those segments to skip parts. By default,
+the app will ask to skip intros and outros, but you can fine-tune this to your liking within the app preferences. All media segment types
+are supported: intro, outro, preview, recap and commercial with two actions to choose from:
+
+- **Skip**
+
+  Immediately skips the playback forward to the end of the segment, or the next episode
+
+- **Ask to skip**
+
+  Shows a popup that the segment can be skipped by pressing the OK button on your remote and disappears after 8 seconds or when pressing the
+  back button.
+
+  ![Ask to skip button shown during video playback](./asktoskip.png)
+
+More actions and functionality related to media segments is already in the works for future releases, like a "mute" action that temporarily
+silences audio.
+
+### Revamped subtitles
+
+This release completely redesigns the way subtitles are loaded and shown on screen, resolving a large number of issues reported over the
+past few years.
+
+Previously the app did not support embedded subtitles and forced the server to extract them from video files first. This is a resource-heavy
+process and often causes subtitles to not show up or with a big delay. A workaround for this was to use the "Subtitle Extract" plugin that
+will do this extraction ahead of time, this was cumbersome and is now no longer necessary.
+
+Together with these loading changes, the app now reliably supports Presentation Graphic Stream (PGS) subtitles and positioning tags in the
+SubRip Subtitle Format (SRT), a much requested feature!
+
+To make these changes happen the preferences for subtitles have also slightly changed. It is now possible to choose your own background and
+outline colors.
+
+### Sing along with lyrics
+
+Not only did video playback improve, playing music has gotten some upgrades as well! The most exciting change here is support for lyrics.
+Whenever your song has lyrics available (added by a plugin or on your file system), they will be shown in the app. This works especially
+well when combined with our screensaver, which will show lyrics fullscreen.
+
+**TODO SCREENSHOT**
```

Comment:
Todo or NotToDo

Reply:
I don't really have content I can use for the screenshot on my server, need  to figure out something for that 👀.

Reply:
Demo server?

Reply:
Unfortunately we don't have any lyrics on the demo server